py-polars/polars/internals/construction.py

```diff
@@ -51,7 +51,6 @@
     _PANDAS_AVAILABLE,
     _PYARROW_AVAILABLE,
     _check_for_numpy,
-    _check_for_pandas,
```

Comment:
I don't think this was intentional.

Reply:
I didn't do it manually, but `make pre-commit` removes it for me

Reply:
I think this should have been caught by https://github.com/pola-rs/polars/actions/runs/4270474083/jobs/7434291441, where the logs say

```
 Found 1 error (1 fixed, 0 remaining).
```
but it didn't fail CI

I'll take a look

Reply:
Hmm... I also saw this change happening locally? Maybe we don't use that line?

Reply:
yup - mystery solved: https://github.com/pola-rs/polars/pull/7215/files
fetching and merging upstream/master fixes it here